src/System.Private.Xml/tests/Writers/XmlWriterApi/TCCloseOutput.cs

```diff
@@ -1,48 +1,191 @@
-// Licensed to the .NET Foundation under one or more agreements.
+﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
 using OLEDB.Test.ModuleCore;
```

Comment:
@danmosemsft, unfortunately not dead yet - I don't remember specifics right now but there were few tiny dependencies left. I can do another pass on those after this is merged. I'd like to get rid of the static state in those tests and finally run them in parallel